<?xml version="1.0" ?>
<launch>
    <node name="localization" pkg="robot_localization" type="ukf_localization_node">
        <param name="world_frame" value="map"/>
        <param name="map_frame" value="map"/>
        <param name="odom_frame" value="odom"/>
        <param name="baselink_frame" value="fcu_local"/>
        <param name="publish_tf" value="true"/>
        <param name="imu0" value="/mavros/imu/data"/>
        <rosparam param="imu0">[False, False, False, True, True, True, False, False, False, True, True, True, True, True, True]</rosparam>
        <param name="odom0" value="/mavros/global_position/local"/>
        <rosparam param="odom0_config">[True, True, True, False, False, False, False, False, False, False, False, False, False, False, False]</rosparam>
        <param name="gps0" value="/mavros/global_position/global"/>
        <rosparam param="gps0_config">[False, False, False, True, True, True, False, False, False, True, True, True, True, True, True]</rosparam>
    </node>
    <node>
        <param name="broadcast_utm_transform" value="true"/>
    </node>
</launch>
